Makefile

```diff
@@ -14,34 +14,48 @@ INSTALL_DIR = $(INSTALL) -d
 INSTALL_PROGRAM = $(INSTALL) -m755
 INSTALL_DATA = $(INSTALL) -m644
 
+ifneq ($(cat /proc/cpuinfo | grep -q ARM),"")
```

Comment:
I am unfamiliar with using an error code like this, but in testing, this check always selects x86 even on an ARM device.

An aside: `grep -q ARM /pro/cpuinfo` would be better.